### PR TITLE
Narsi plushie can invoke now

### PR DESCRIPTION
**Pull request by SuperSlayer0**
## About The Pull Request

e

## Why It's Good For The Game

e

## Changelog
:cl:
add: Narsie plushie can now invoke cult runes again
/:cl:

### CODE REVIEW

**Comment by Pugie** [comment]
a great wrong has been corrected

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -485,6 +485,11 @@
 		qdel(src)
 		P.clashing = FALSE
 
+/obj/item/toy/plush/narplush/hugbox
+	desc = "A small stuffed doll of the elder goddess Nar'Sie. Who thought this was a good children's toy? <b>It looks sad.</b>"
+	is_invoker = FALSE
+	var/is_invoker = TRUE
```
Comment (TheChosenEvilOne) [inline code/game/objects/items/plushes.dm:491]: guh?

Comment (SuperSlayer0) [reply code/game/objects/items/plushes.dm:491]: From old code, there is a subtype of narsie toy that cannot invoke and it has this var on false.

Comment (SuperSlayer0) [reply code/game/objects/items/plushes.dm:491]: Fuck

Comment (SuperSlayer0) [reply code/game/objects/items/plushes.dm]: Fixed

**Pull request closed without merging**